uberenv.py

```diff
@@ -1065,12 +1065,13 @@ def install(self):
                     if len(hc_glob) > 0:
                         hc_path  = hc_glob[0]
                         hc_fname = os.path.split(hc_path)[1]
-                        if os.path.islink(hc_fname):
-                            os.unlink(hc_fname)
-                        elif os.path.isfile(hc_fname):
-                            sexe("rm -f {0}".format(hc_fname))
-                        print("[symlinking host config file to {0}]".format(pjoin(self.dest_dir,hc_fname)))
-                        os.symlink(hc_path,hc_fname)
```

Comment:
this guy was the issue, it wasn't use the pjoined path that was actually printed

Reply:
my mistake. thank you for catching this